pom.xml

```diff
@@ -0,0 +1,356 @@
+        <magnolia.site.version>1.4</magnolia.site.version>
+
+        <source.javaversion>11</source.javaversion>
+        <target.javaversion>11</target.javaversion>
+
+        <source.encoding>UTF-8</source.encoding>
+        <project.build.sourceEncoding>${source.encoding}</project.build.sourceEncoding>
+
+        <!-- checkstyle can be disabled with this property -->
+        <checkstyle.skip>false</checkstyle.skip>
+        <checkstyle.config>src/config</checkstyle.config>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>info.magnolia.ui</groupId>
+                <artifactId>magnolia-ui-admincentral</artifactId>
+                <version>${magnoliaVersion}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>commons-lang</groupId>
+                        <artifactId>commons-lang</artifactId>
+                    </exclusion>
+                    <!-- get rid of transitive missing vaadin-server.rc1 pom -->
+                    <exclusion>
+                        <groupId>com.vaadin.addon</groupId>
+                        <artifactId>date-range-field</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>info.magnolia.core</groupId>
+                <artifactId>magnolia-virtual-uri</artifactId>
+                <version>${magnoliaVersion}</version>
+            </dependency>
+            <dependency>
+                <groupId>info.magnolia.types</groupId>
+                <artifactId>magnolia-content-types</artifactId>
+                <version>${magnolia.content-types.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>info.magnolia.site</groupId>
+                <artifactId>magnolia-site</artifactId>
+                <version>${magnolia.site.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>info.magnolia.site</groupId>
+                <artifactId>magnolia-site-app</artifactId>
+                <version>${magnolia.site.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>javax.servlet</groupId>
+                <artifactId>servlet-api</artifactId>
+                <version>2.5</version>
+                <scope>provided</scope>
+            </dependency>
+            <dependency>
+                <groupId>javax.servlet.jsp</groupId>
+                <artifactId>jsp-api</artifactId>
+                <version>2.1</version>
+                <scope>provided</scope>
+            </dependency>
+
+            <!-- TEST -->
+            <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>4.13.1</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
```

Comment:
I believe with the new Magnolia version a new Mockito version is supported.

Reply:
done!